strawberryfields/apps/graph/utils.py

```diff
@@ -169,14 +180,27 @@ def c_0(clique: list, graph: nx.Graph):
     """Generates the set :math:`C_0` of nodes that are connected to all nodes in the input
     clique subgraph.
 
-    The set :math:`C_0` is defined in :cite:`pullan2006phased`.
+    The set :math:`C_0` is defined in :cite:`pullan2006phased` and is used to determine nodes
+    that can be added to the current clique to grow it into a larger one.
+
+    Example usage:
+
+    .. code-block::
+
+        >>> from strawberryfields.apps.graph import utils
+        >>> import networkx as nx
+        >>> graph = nx.complete_graph(10)
+        >>> subgraph = [0, 1, 2, 3, 4]
+        >>> utils.c_0(subgraph, graph)
+        [5, 6, 7, 8, 9]
 
     Args:
-        clique (list[int]): A subgraph specified by a list of nodes; the subgraph must be a clique
-        graph (nx.Graph): the input graph
+        clique (list[int]): A subgraph specified by a list of nodes; the subgraph must be a clique.
```

Comment:
```suggestion
        clique (list[int]): a subgraph specified by a list of nodes; the subgraph must be a clique
```

Reply:
^ We need to check for consistency throughout the documentation regarding upper cases and full stops